src/PrestaShopBundle/Install/Install.php

```diff
@@ -704,14 +704,14 @@ public function copyLanguageImages($iso)
 
     public function getLocalizationPackContent($version, $country)
     {
-        if (static::$_cache_localization_pack_content === null || array_key_exists($country, static::$_cache_localization_pack_content)) {
+        if (self::$_cache_localization_pack_content === null || array_key_exists($country, self::$_cache_localization_pack_content)) {
```

Comment:
Please restore static instead of self.

Reply:
No! If property is declared as `private`, `static::` access will fail if the class is extended.